app/assets/javascripts/samples/sample_datatable.js

```diff
@@ -763,4 +754,87 @@ function changeToEditMode() {
 
     // Table specific stuff
     table.button(0).enable(false);
-}
\ No newline at end of file
+}
+
+(function(table) {
+  'use strict';
+
+  var dropdownBtn = $('#samples-columns-dropdown');
+  var dropdownList = $('#samples-columns-list');
+
+  // removes button created by dataTables
+  function removeDefaultButton() {
```

Comment:
We can just remove default buttons from table definition, so we don't need to remove them lately